### PR TITLE
Block reward address and Conversion rate between space pledged and solution range

### PR DESCRIPTION
**Pull request by xm0onh**
This PR consists of two paragraphs:
1. Describing the Block Reward Address on the Consensus chain page.
2. Conversion rate between solution range and space pledged (#13 )

### CODE REVIEW

File: docs/consensus/consensus_chain.md [hunk 1]
```diff
@@ -125,3 +125,25 @@ Justifications contain a set of all PoT checkpoints since the parent block up to
...
+
+## Block Reward Address
+In the current blockchain farming setup, a farmer’s identity is used for plot creation, block signing, and receiving block rewards, posing risks like plot invalidation on testnet when used across multiple nodes and incompatibility with cold wallets. To address these, it's proposed to decouple the farmer's identity from the reward address by introducing a `--reward-address` argument in the farmer app, allowing the specification of a separate address for block rewards. This change, reflected in the updated `Solution` structure with an additional `reward_address` field, would enhance security, support multi-replica farming, and align farming operations with practices in PoW mining where reward addresses are independent of operational identities.
+
+```rust
+pub struct Solution<AccountId> {
+    /// Public key of the farmer that created the solution
+    pub public_key: AccountId,
++    /// Address for receiving block reward
++    pub reward_address: AccountId,
```
Comment (dariolina) [inline docs/consensus/consensus_chain.md:137]:
```suggestion
    /// Address for receiving block reward
    pub reward_address: AccountId,
```

File: docs/consensus/consensus_chain.md [hunk 2]
```diff
@@ -125,3 +125,25 @@ Justifications contain a set of all PoT checkpoints since the parent block up to
...
+    /// Index of encoded piece
+    pub piece_index: u64,
+    /// Encoding
+    pub encoding: Piece,
```
Comment (dariolina) [inline docs/consensus/consensus_chain.md:141]: Actually these `struct` fields are outdated. In the `proof_of_archival_storage.md` spec somewhere there should be the new one. Or double-check it with the codebase directly.

File: docs/consensus/consensus_chain.md [hunk 3]
```diff
@@ -125,3 +125,25 @@ Justifications contain a set of all PoT checkpoints since the parent block up to
 3. Import and verify blocks. Ban bad peers.
 4. When you are close to tip (~18 blocks) switch to keep-up sync
 5. When get to tip start participate in consensus
+
+## Block Reward Address
+In the current blockchain farming setup, a farmer’s identity is used for plot creation, block signing, and receiving block rewards, posing risks like plot invalidation on testnet when used across multiple nodes and incompatibility with cold wallets. To address these, it's proposed to decouple the farmer's identity from the reward address by introducing a `--reward-address` argument in the farmer app, allowing the specification of a separate address for block rewards. This change, reflected in the updated `Solution` structure with an additional `reward_address` field, would enhance security, support multi-replica farming, and align farming operations with practices in PoW mining where reward addresses are independent of operational identities.
```
Comment (dariolina) [inline docs/consensus/consensus_chain.md:130]:
```suggestion
In a basic blockchain farming setup, a farmer’s identity would be used for plot creation, block signing, and receiving block rewards, posing risks like plot invalidation on testnet when used across multiple nodes and incompatibility with cold wallets. To address these,  the farmer's identity is decoupled from the reward address by introducing a `--reward-address` argument in the farmer app, allowing the specification of a separate address for block rewards. This  additional `reward_address` block header field, enhances security, supports multi-replica farming, and aligns farming operations with practices in PoW mining where reward addresses are independent of operational identities.
```

**Review by dariolina — CHANGES_REQUESTED** [review]

**Comment by xm0onh** [comment]
Thanks!
I made the changes and added the updated struct from the subspace repo: https://github.com/subspace/subspace/blob/f8bf2809a565d3044623838a08e7417eade95168/crates/subspace-core-primitives/src/lib.rs#L603

**Review by dariolina — APPROVED** [review]

**Pull request merged**